Game/Source/GDKTestGyms/Private/BenchmarkNPCCharacter.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
```

Comment:
Minor, double new lines in this file at start, after includes and end seem strange. Surprised our linter didn't pick this up

Reply:
If that's something I need to set up locally it's likely I haven't done it yet :P This is just what the file had when Unreal created it for me.